Header header
int32 ac_id
geometry_msgs/Point position
Attitude attitude
float64 battery_level
